dotnet/src/SemanticKernel/SkillDefinition/SKFunction.cs

```diff
@@ -85,6 +83,37 @@ public CompleteRequestSettings RequestSettings
             log: log);
     }
 
+    /// <summary>
+    /// Create a native function instance, wrapping a delegate function
+    /// </summary>
+    /// <param name="nativeFunction">Function to invoke</param>
+    /// <param name="skillName">SK skill name</param>
+    /// <param name="functionName">SK function name</param>
+    /// <param name="description">SK function description</param>
+    /// <param name="parameters">SK function parameters</param>
+    /// <param name="log">Application logger</param>
+    /// <returns>SK function instance</returns>
+    public static ISKFunction FromNativeFunction(
```

Comment:
Remember, we are dropping the term NativeFunction. 
FromDelegate?

Reply:
we can have that conversation offline